cypress/e2e/cloud/annotations.test.ts

```diff
@@ -60,7 +62,12 @@ describe('The Annotations UI functionality', () => {
     // cy.clearLocalStorage only clears for current domain/subdomain.
     // Putting it here so it clears the data at this subdomain consistently.
     // https://github.com/cypress-io/cypress/issues/2573
-    cy.clearLocalStorage()
+    // cy.clearLocalStorage()
```

Comment:
cut this commented out line.

is the comment above it still relevant and valid? if so, keep it, if not, cut it.

Reply:
lets leave the comment with the issue link please! Feel free to update the text above the link but I think the link is still useful.

Reply:
For sure I can add the link back!